server/src/main/java/io/druid/segment/indexing/granularity/ArbitraryGranularitySpec.java

```diff
@@ -46,34 +45,25 @@
   private final TreeSet<Interval> intervals;
   private final QueryGranularity queryGranularity;
   private final Boolean rollup;
-  private final String timezone;
 
   @JsonCreator
   public ArbitraryGranularitySpec(
       @JsonProperty("queryGranularity") QueryGranularity queryGranularity,
       @JsonProperty("rollup") Boolean rollup,
-      @JsonProperty("intervals") List<Interval> inputIntervals,
-      @JsonProperty("timezone") String timezone
```

Comment:
No, his response is saying that it's backwards incompatible, but it's backwards incompatible with something that only exists in master, not in any released code.  So, it won't actually be backwards incompatible as far as the release is concerned.